PayPal/src/main/java/com/braintreepayments/api/PayPalAnalytics.kt

```diff
@@ -0,0 +1,14 @@
+package com.braintreepayments.api
+
+internal enum class PayPalAnalytics(@JvmField val event: String) {
+
+    // Conversion Events
+    TOKENIZATION_STARTED("paypal:tokenize:started"),
+    TOKENIZATION_FAILED("paypal:tokenize:failed"),
+    TOKENIZATION_SUCCEEDED("paypal:tokenize:succeeded"),
+    BROWSER_LOGIN_CANCELED("paypal:tokenize:browser-login:canceled"),
+
+    // Browser Presentation Events
+    BROWSER_SWITCH_SUCCEEDED("paypal:tokenize:browser-presentation:succeeded"),
+    BROWSER_SWITCH_FAILED("paypal:tokenize:browser-presentation:failed")
```

Comment:
[On iOS (https://github.com/braintree/braintree_ios/blob/74f056ac0600c48bdb2c4908075fcf5580a71a86/Sources/BraintreePayPal/BTPayPalClient.swift#L293-L298) we have a synchronous `bool` return value from ASWeb into if the ASWeb view presented or didn't.

Do we have this insight on Android? A way to know if the CCT displayed or didn't?

Reply:
[`BrowserSwitchClient#start`](https://github.com/braintree/browser-switch-android/blob/main/browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java#L47) (which launches the CCT) throws a `BrowserSwitchException` in some cases, mainly for improper deep linking configuration, but also if the host activity is finishing while trying to launch CCT. In those cases if we receive that `BrowserSwitchException`, we would send that `BROWSER_SWITCH_FAILED` analytic event on Android. So it's not a 1 to 1 match with the iOS meaning of that event, but I think it's still useful for Android

Reply:
Awesome, this is great - thanks for the explanation